frontend/js/views/timeline.js

```diff
@@ -73,22 +73,38 @@ define([
         item.group = annotation.collection.track.id;
         item.start = util.dateFromSeconds(item.start);
         item.end = util.dateFromSeconds(item.end);
-        var label = item.label;
-        if (label) {
-            var color = annotation.color();
-            item.style =
-                "background-color:" + color + ";" +
+        delete item.content;
+
+        item.text = annotation.get("content")
+            .chain()
+            .sortBy(function (c) { return c.get("type"); })
+            .invoke("getText")
+            .value()
+            .join(", ");
+
+        var color = annotation.getColor();
+        item.style = color && (
+            "background-color:" + color + ";" +
                 "color:" + (
                     chroma(color).luminance() < 0.5
-                        ? "white"
+                        ? "black" // before: white
```

Comment:
Colors wouldn't work otherwise with MCA

Reply:
Does MCA stand for "Multi Colored Annotation"? Is that an annotation with multiple labels of differing colors?

This code makes the conditional redudant, as the color will now always be black. Which coincidentally works, as none of the predefined category colors are dark enough to make black unreadable I suppose (at least that's what I think this is about?). Relying on the predefined category colors not changing seems suboptimal though.

Reply:
[Multi-Content-Annotation](https://github.com/opencast/annotation-tool/labels/mca)
(previously: annotation was either "free text" or "label", now: annotation can contain multiple "free text" and/or "labels")

I remember there was a reason with this change regarding colors of MCA and display style of annotations in timeline;
I think the conditional was kept with the comment as it was unsure whether this would require another change later on
we could figure out what value(s) work in all (or most) cases and adjust it again

Reply:
Thanks for the explanation. I looked into it a bit, and unfortunately there seems to be no quick fix for this, at least I am unable to come up with one. Since this code works with the current state of the annotation tool, albeit coincidentally, I think we can leave this as is for now (I did add a todo to the code though).